src/var_store.rs

```diff
@@ -80,36 +80,55 @@ impl<'a> ItemVariationStore<'a> {
         let offset = self.data_offsets.get(outer_index)?;
         let mut s = Stream::new_at(self.data, usize::num_from(offset))?;
         let item_count = s.read::<u16>()?;
-        let short_delta_count = s.read::<u16>()?;
+        let word_delta_count = s.read::<u16>()?;
         let region_index_count = s.read::<u16>()?;
         let region_indices = s.read_array16::<u16>(region_index_count)?;
 
         if inner_index >= item_count {
             return None;
         }
 
-        let delta_set_len = usize::from(short_delta_count) + usize::from(region_index_count);
-        s.advance(usize::from(inner_index).checked_mul(delta_set_len)?);
+        let has_long_words = (word_delta_count & 0x8000) != 0;
+        let word_delta_count = word_delta_count & 0x7FFF;
+
+        let delta_set_len = if has_long_words {
+            4 * word_delta_count + 2 * region_index_count.checked_sub(word_delta_count)?
+        } else {
+            2 * region_index_count
+        };
+        s.advance(usize::from(inner_index).checked_mul(usize::from(delta_set_len))?);
 
         let mut delta = 0.0;
         let mut i = 0;
-        while i < short_delta_count {
+        while i < word_delta_count {
             let idx = region_indices.get(i)?;
-            delta += f32::from(s.read::<i16>()?) * self.regions.evaluate_region(idx, coordinates);
+            let num = if has_long_words {
+                // TODO: use f64?
+                s.read::<i32>()? as f32
```

Comment:
Not sure if we should switch to f64 here, because in theory it could overflow...

Reply:
It's fine for now. Let's leave it for the future.